expression-language/5.0/_index.md

```diff
@@ -44,21 +44,23 @@ The following TCK updates will be required:
 
 ## Plan Review
 
-The Plan Review Specification Committee Ballot concluded successfully on TBD with the following results.
-
-|                       |  Yes    | No  | Abstain  |
-|-----------------------|---------|-----|----------|
-|Fujitsu                |         |     |          |
-|IBM                    |         |     |          |
-|Oracle                 |         |     |          |
-|Payara                 |         |     |          |
-|Red Hat                |         |     |          |
-|Tomitribe              |         |     |          |
-|EE4J PMC               |         |     |          |
-|Participant Members    |         |     |          |
-|Committer Members      |         |     |          |
-
- This ballot was conducted on the public e-mail list [jakarta.ee-spec@eclipse.org](). This ballot thread begins [here](TBD).
+The Plan Review Specification Committee Ballot concluded successfully on 12/05/2021 with the following results.
```

Comment:
I know I'm being picky and probably US-centric, @Pandrex247, but can you follow the mm/dd/yyyy convention, or spell out the Month?  My first reaction to this was to read this as Dec 05 and that would be difficult to predict... :-)  Thanks!

Reply:
Muscle memory 😜
I've clarified it

Reply:
A bit more elaborate change than I expected, but this works...  ;-)  Thanks!